src/components/ButtonNav/index.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import { ButtonLink, layoutType } from '../Button'
+import { Wrapper } from './styled'
+import mentor from '../../assets/mentor.svg'
+import buddy from '../../assets/buddy.svg'
+import mentee from '../../assets/mentee.svg'
+
+export const ButtonNav: React.FC = () => {
+	return (
+		<Wrapper>
```

Comment:
Tohle bude ještě vtipné, až se budeme snažit mít víc druhů buttonů s různými texty. :joy: Nějakým podmínkám se pak asi nevyhneme. :thinking: A nebo budeme mít další soubor pro Nav před registrací a další pro Nav po registraci a pak tenhle pro ten specifický případ, kdy tam bude Mentor, Mentee a Buddy.

Reply:
No zatím je to nastavené tak, že si můžeme vybrat tlačítko s funkcí nebo s odkazem, které je buď zarovnané do středu pod sebe nebo do řádku vedle sebe  (obsah uvnitř) . A můžeme u něj vybrat zda chceme ikonu, nadpis nebo text (možno vše dohromady). A Wrapper má zastávat funkci nejen obalu, ale i dávat rozložení tlačítek na stránce a jejich případnou velikost. Nechtěla jsem dát fixní velikost tlačítkům, přijde mi lepší, když to ovlivňuje prostředí, ve kterém se nachází 🙂

A ano, Wrapperů tam bude asi hodně, ale řeší nám problém s možnostmi tlačítek. Pokud jsem to teda napsala dobře 😅